dns-server/tests/basic_test.rs

```diff
@@ -479,3 +562,36 @@ async fn dns_records_list(
         .map(|z| z.records)
         .unwrap_or_else(HashMap::new))
 }
+
+/// issue a DNS query of `record_ty` records for `name`.
```

Comment:
nit: I (we?) use traditional capitalization.  I see a lot of this file already doesn't use caps or punctuation, but that seems to be mostly the older one-line comments.